src/app/components/typography/TypographyTable.tsx

```diff
@@ -109,14 +109,25 @@ export const TypographyTable = (): JSX.Element => {
         [theme]
     );
 
+    const formatName = useCallback((variant: TypographyVariant): string => {
+        if (variant.startsWith('h')) {
+            const num = variant.charAt(1);
+            if (num === '1') {
+                return `HeadL1`;
```

Comment:
:(
"HeadL1" is confusing. Rather see H1 here.

Reply:
It shall be H1